install/resources/settings.xml

```diff
@@ -2418,6 +2418,16 @@ delete=Delete]]></optionscode>
 			<isdefault>1</isdefault>
 			<helpkey></helpkey>
 		</setting>
+		</setting>
+		<setting name="purgespammerbanip">
+			<title>Add banned IP filter</title>
+			<description><![CDATA[Do you want to ban the spammer's IP using this tool? If other users are issued the same banned IP, they will not be able to use the forums.]]></description>
+			<disporder>6</disporder>
```

Comment:
This should be 7, after this it should be fine to be merged. Eventually we should get rid of these unnecessary elements.
```suggestion
			<disporder>7</disporder>
```